lib/xcode/install/simulators.rb

```diff
@@ -42,7 +44,7 @@ def install
         simulator = filtered_simulators.first
         fail Informative, "#{simulator.name} is already installed." if simulator.installed? && !@force
         puts "Installing #{simulator.name} for Xcode #{simulator.xcode.bundle_version}..."
-        simulator.install(@progress, @should_install)
+        simulator.install(@progress, @should_install, @retry_download_count.to_i)
```

Comment:
it would be bettter to validate string to number before passing to `install`, or it will show complex ruby error if string is not valid for parsing number
```suggestion
        fail Informative, "Invalid Retry: `#{@retry_download_count} is not positive number.`" if (@retry_download_count =~ /\A[0-9]*\z/).nil?
        simulator.install(@progress, @should_install, @retry_download_count.to_i)
```